src/libs/OptionsListUtils.js

```diff
@@ -719,7 +719,7 @@ function getHeaderMessage(hasSelectableOptions, hasUserToInvite, searchValue, ma
         return translate(preferredLocale, 'messages.noPhoneNumber');
     }
 
-    if (!hasSelectableOptions && !hasUserToInvite) {
+    if (searchValue && !hasSelectableOptions && !hasUserToInvite) {
```

Comment:
@tgolen
While using this method for the split I found a case where I didn't give any search input just picked users from the recent list. Since users are picked are excluded from `personalDetails`, the `personalDetails` list become empty that `hasSelectableOptions` is false so `No result found` is returned. So I added a check that `searchValue` should be available with the existing checks.

This additionally added along proceed with proposed changes. If you are good with this change, I'll test it once again and mark it ready for review!

Reply:
I think that's fine, yeah. Thank you for the heads up.

Reply:
Would you mind adding a comment to the code to explain why `searchValue` needs to be in that logic? I just don't want someone else to come along and see it and think it's OK to remove it without understanding the implications of it being there.

Reply:
In short I think -> `Without a searchValue should not return any search validation messages`
Is this is okay?
Do you have anything specific comment in mind @tgolen?

Reply:
Yeah, that's pretty good! I would expand it a little further like this:

> Without a search value, it would be very confusing to see a search validation message. Therefore, this skips the validation when there is no search value.

Reply:
> Without a search value, it would be very confusing to see a search validation message. Therefore, this skips the validation when there is no search value.

This is better, updated the comment!